commit ccad070c2f421cf5f81a732a9538620f74399470 (refs/changes/35/735/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-19T11:31:36-07:00 (3 years ago)
    
    IMGAPI-596 update imgapi and docker to use restify-clients@1.4.0 with upstreamed http_proxy support
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/package.json b/package.json
index 0514975..4fe70df 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
         "bunyan": "1.x >=1.3.3",
         "jws": "3.1.0",
         "jwk-to-pem": "1.2.0",
-        "restify-clients": "git+https://github.com/trentm/clients.git#0b989e0bbd375ddeefce2f6e2754f66e5dbb59d1",
+        "restify-clients": "^1.4.0",
         "restify-errors": "^3.0.0",
         "strsplit": "1.x",
         "tough-cookie": "2.0.x",
